packages/cudos-cmd/keys/keys.js

```diff
@@ -0,0 +1,55 @@
+const VError = require('verror')
+
+const {
+  executeNode,
+  executeNodeMultiCmd
+} = require('../../cudos-utilities/run-docker-commands')
+
+const keysListCmd = async function() {
+  try {
+    executeNode('keys list')
+  } catch (error) {
+    console.log("Could not fetch keys, is your node online? Execute 'cudo node status' for more info")
+  }
+}
+
+const keysAddCmd = async function(argv) {
+  try {
+    executeNodeMultiCmd(`cudos-noded keys add ${argv.name} && ` + transferTokensByNameCommand(
+      'faucet', argv.name, '1000000000000000000'))
+  } catch (error) {
+    throw new VError(`Could not add account ${argv.name}, \nError: ${error.message}`)
+  }
+}
+
+const keysRmCmd = async function(argv) {
+  try {
+    if (argv.yes) {
+      executeNode(`keys delete ${argv.name} --yes`)
+    } else {
+      executeNode(`keys delete ${argv.name}`)
```

Comment:
1.can we remove the `else` and jsut return in the `if`.
2. If the argument is not `yes` why we are still removing the account ?

Reply:
1. The idea is to have a confirmation when deleting a key. Whether we want such feature is up for a discussion.
2. Do you mean that if the tool encounters unknown optional parameter it should stop?